results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+
+### UI customization
+
+- **Participants cared about themes.** Of participants who immediately changed settings to make JupyterLab better for their needs, all but one took the time to change the theme. Participants who were not aware JupyterLab had multiple default themes and found out later in the session remarked positively.
+- **Adjusting all text’s font size and having JupyterLab subsequently reflow its content was highly desired.** This is beyond default JupyterLab’s current text adjustment capabilities, but it does mean that the font size settings we have now are on the right track. 
+- **[Low color contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) was not a major blocker for participants to complete tasks, but it was unpleasant.** Some participants overwrote sections of JupyterLab’s default colors with their browser to remedy it, but that is specialized knowledge not available to all possible users. The biggest issues arose when reading code; the syntax highlighting had particularly poor contrast or color combinations that were not distinct to participants.
+
+### Notebook
+
+- **Notebook structure and content does impact participant experience.** This means authors have the power to make working in JupyterLab more accessible even before changes can be made to JupyterLab itself. For our sessions, we received positive feedback on the [NumPy Tutorial notebook](https://numpy.org/numpy-tutorials/content/save-load-arrays.html)). Kudos to the NumPy Tutorials and wider documentation team!
+- **This study did not test a long or mixed content notebook.** Participants mentioned that there was nothing in this notebook they were wholesale not able to access, though the default styling sometimes made it harder to read. We did not test a notebook with images, plots, or widgets, which in other notebook interfaces have proven difficult.
+- **Markdown headings were indispensable in keeping participants and facilitators in the same place.**
+- **The width of a document, notebook or otherwise, and length of code lines impacts magnification users.** Wider or longer sections require more horizontal movement. This is taxing and increases the likelihood of missing information. 
+- A majority of participants were more familiar with different notebook interfaces than JupyterLab. **The notebook file was the common ground for all.** Participants expected the notebook to behave the same regardless of interface.
+
+## Comparing assistive tech and non-assistive tech experiences
+
+Because we had participants who were using assistive tech and participants were not using assistive tech, we were able to observe some of the user experience differences between the two groups. Many of these comparisons highlight some of the disparities and impacts of having an inaccessible interface.
```

Comment:
```suggestion
Because we had participants who were using assistive tech and participants who were not using assistive tech, we could observe some of the user experience differences between the two groups. Many of these comparisons highlight the disparities and impacts of having an inaccessible interface.
```